src/main/resources/graphql/service/mutation.graphqls

```diff
@@ -153,6 +182,210 @@ input MultipleChoiceAnswerInput {
     feedback: String @Size(max: 1000)
 }
 
+input CreateClozeQuestionInput {
+    """
+    Number of the question, used for ordering.
+    This can be omitted, in which case a number, one higher than the highest number of the existing questions, will be used.
+    """
+    number: Int @Positive
+    """
+    Text of the question, can be markdown.
+    """
+    text: String! @Size(max: 1000)
+    """
+    List of cloze elements.
+    """
+    clozeElements: [ClozeElementInput!]! @ContainerSize(min: 1)
+    """
+    Optional hint for the question, can be markdown.
+    """
+    hint: String @Size(max: 1000)
+}
+
+input UpdateClozeQuestionInput {
+    """
+    UUID of the question to update.
+    """
+    id: UUID!
+    """
+    Text of the question, can be markdown.
+    """
+    text: String! @Size(max: 1000)
+    """
+    List of cloze elements.
+    """
+    clozeElements: [ClozeElementInput!]! @ContainerSize(min: 1)
+    """
+    Optional hint for the question, can be markdown.
+    """
+    hint: String @Size(max: 1000)
+}
+
+input ClozeElementInput {
+    """
+    Type of the element.
+    """
+    type: ClozeElementType!
+
+    """
+    Text of the element, can be markdown. Only used for TEXT type.
+    """
+    text: String! @Size(max: 1000)
+
+    """
+    The correct answer for the blank. Only used for BLANK type.
+    """
+    correctAnswer: String!
+    """
+    Wrong answers for the blank. Only used for BLANK type.
+    """
+    wrongAnswers: [String!]!
```

Comment:
Verstehs nicht ganz. Haben wir eine Lücke mit mehreren Antwortmöglichkeiten, wo eine richtig und die anderen falsch sind oder haben wir wirklich einen Lückentext und für jede Lücke im Text erstellt man mehrere falsche und eine richtige Antwort?

Reply:
hm yeah maybe assigning wrong answers to each gap is overkill. I just though that you have a pool of strings you can pick from, but not all of them belong to a gap (wrong answers). So maybe this can be moved to the question.